chipy_org/apps/meetings/serializers.py

```diff
@@ -4,7 +4,7 @@
 
 
 class PresentorSerializer(serializers.ModelSerializer):
-    email = serializers.SerializerMethodField('get_email')
```

Comment:
Thanks. I haven't touched Rest Framework 3. Good to know.

Weird it throws an `AssertionError` though. Seems like that would be much more suited to a warning.